src/components/send/amount-field.tsx

```diff
@@ -1,84 +0,0 @@
-import { Box, color, Input, InputGroup, Stack, StackProps, Text } from '@stacks/ui';
```

Comment:
This file was not actually deleted, but due to the way it was moved, git thinks it was. Can you make it so the history is preserved?